core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -563,11 +570,13 @@ public UnderFileSystem getUnderFileSystem() {
    */
   private <T> T call(UfsCallable<T> callable) throws IOException {
     LOG.debug("Enter: {}", callable);
-    try {
+    String methodName = callable.toString().split(NAME_SEPARATOR)[0];
+    try (Timer.Context ctx = MetricsSystem.workerTimer(getQualifiedMetricName(methodName)).time()) {
       T ret = callable.call();
       LOG.debug("Exit (OK): {}", callable);
       return ret;
     } catch (IOException e) {
+      MetricsSystem.workerCounter(getQualifiedFailureMetricName(methodName)).inc();
```

Comment:
Why only increment upon exception but not before the try block?

Reply:
I'm keeping a separate metric which tracks the number of failures. The timer should take care of the number of calls.

Reply:
hmm the timer will increment the number of calls? If so, then we should be consistent in your the other PR https://github.com/Alluxio/alluxio/pull/7367/files#diff-d7dc60da6f1005850b481cc5792912c5R314

Reply:
That call is to count the retries. We do not retry UFS operations.

Reply:
Got it. Thanks.